main.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+#include "Candidate.h"
+#include "Elections.h"
+int main() {
+    using namespace E;
+    Elections elections;
+    elections.setCandidates();
+    elections.countRates();
+    elections.printAll();
+    elections.whoWins();
+
```

Comment:
Please change to
elections.set_candidates();
    elections.count_rates();
    elections.print_all();
    elections.who_wins();